craft_application/models/project.py

```diff
@@ -64,7 +83,9 @@ class Project(CraftBaseModel):
     summary: Optional[SummaryStr]
     description: Optional[str]
 
-    base: Optional[Any]
+    base: Optional[Any] = None
+    build_base: Optional[Any] = None
```

Comment:
where are these new fields used?

Reply:
I was thinking they are used in the `craft-grammar`, but seems not? @cmatsuoka